### PR TITLE
Luk10114/83 set min max scale

### PR DESCRIPTION
**Pull request by LDuncAndroid**
https://devtopia.esri.com/runtime/common-samples/issues/83

### CODE REVIEW

File: java/set-min-max-scale/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.samples.setminxmaxscale">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <uses-feature
+        android:glEsVersion="0x00020000"
```
Comment (TADraeseke) [inline java/set-min-max-scale/src/main/AndroidManifest.xml]: Samples using 3D (scenes) require Open GL ES 3.0. Maps requires 2.0, but uses 3.0 if available. May need to check this on Integrated Mesh sample, as I may have forgotten to highlight it to you.

File: kotlin/set-min-max-scale/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.samples.setminxmaxscale">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <uses-feature
+        android:glEsVersion="0x00020000"
```
Comment (TADraeseke) [inline kotlin/set-min-max-scale/src/main/AndroidManifest.xml]: as above

File: java/set-min-max-scale/src/main/AndroidManifest.xml (hunk 1 above)
Comment (LDuncAndroid) [reply java/set-min-max-scale/src/main/AndroidManifest.xml]: This sample is just 2D.

File: kotlin/set-min-max-scale/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Min Max Scale
+
+Restrict zooming to a specific scale range.
+
+![Min Max Scale](min-max-scale.png)
+
+## How it works
+
+1. Create an ArcGIS map.
+1. Set min and max scales of map, ```ArcGISMap.setMaxScale()``` and ```ArcGISMap.setMinScale()```.
```
[the remark this answers is not in the document]
Comment (LDuncAndroid) [reply kotlin/set-min-max-scale/README.md]: Yeah, my bad.

**Comment by TADraeseke** [comment]
Looks good! Thanks @LDuncAndroid

**Pull request merged**